Add webpage-to-markdown extension

treyg opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Would it make positible to show an error if its not a webpage?

![webpage-to-markdown 2024-11-04 at 16 02 38](https://github.com/user-attachments/assets/384ed309-a854-42f2-80b1-f7b32d40b0ca)

- You can use images from `metadata` in your `README`

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

treyg commented:
@pernielsentikaer Thanks for the suggestions, and good catch! I added some error handling so you'll now see this if something other than a webpage url is posted: 
![webpage-to-markdown 2024-11-05 at 12 24 34](https://github.com/user-attachments/assets/d26cd97c-6d64-4f5d-8f89-ebcefe304969)

I also removed the duplicated images from the assets folder and am just using the reference to the metadata for the readme. Sorry, I was a bit confused by the "assets used by the README are placed outside of the metadata folder" checkbox upon publish. 

Thanks again!

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
